packages/kbn-investigation-shared/src/rest_specs/create_item.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as t from 'io-ts';
+import { investigationItemsSchema } from '../schema';
+import { investigationItemResponseSchema } from './investigation_item';
+
+const createInvestigationItemParamsSchema = t.type({
+  path: t.type({
+    investigationId: t.string,
+  }),
+  body: investigationItemsSchema,
+});
+
+const createInvestigationItemResponseSchema = investigationItemResponseSchema;
+
+type CreateInvestigationItemInput = t.OutputOf<
+  typeof createInvestigationItemParamsSchema.props.body
+>;
+type CreateInvestigationItemParams = t.TypeOf<
```

Comment:
I was trying to understand the difference between `CreateInvestigationItemParams` and `CreateInvestigationItemInput` and I didn't find `CreateInvestigationItemParams` being used anywhere. Can you explain the difference?

Reply:
I will remove the Params one since they are not used. 
Basically it's what I explained in another comment about the DateString <-> Date object.
OutputOf would be the type with the Date, while TypeOf would be the type with the String representation of the Date.